src/uwtools/drivers/driver.py

```diff
@@ -25,7 +25,7 @@
 from uwtools.scheduler import JobScheduler
 from uwtools.strings import STR
 from uwtools.utils.file import writable
-from uwtools.utils.processing import execute
+from uwtools.utils.processing import shellcmd
```

Comment:
I feel a little like I'm channeling Mike Kay for this one, but we moved from a verb to a noun for a function. I noticed a log message below moved from "Executing..." to "Running..." Could we consider `runshellcmd` for this one?

Reply:
Good point. `runshellcmd` strikes me as somewhat clunky -- how about `run_shell_cmd`?

Reply:
I went ahead with `run_shell_cmd`. If anyone hates it we can easily change it later -- we can now do an unambiguous search-and-replace for this function name since it doesn't conflict with `execute`.